### PR TITLE
Update nio_udp.c

### PR DESCRIPTION
**Pull request by sebasdoes**
Adding this part fixes FreeBSD support. Only tested on FreeBSD.

### CODE REVIEW

File: nio_udp.c [hunk 1]
```diff
@@ -25,6 +25,7 @@
 #include <arpa/inet.h>
 #include <limits.h>
 #include <netdb.h>
+#include <netinet/in.h>
```
Comment (noplay) [inline nio_udp.c]: I think we could add a condition in order to include only on BSD.

Comment (grossmj) [inline nio_udp.c]: This should compile on Linux. Not sure about Cygwin.

I would check the above first and if this doesn't work then add a condition like:

```
#ifdef __FreeBSD__
  #include <netinet/in.h>
#endif
```

**Comment by grossmj** [comment]
I have checked, this does not conflict with Linux and Windows compilations.

**Comment by grossmj** [comment]
Thanks! :+1:

**Pull request merged**